lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -618,7 +618,7 @@ def add_subscription(xml, options, reference = nil)
 
         xml.tag! 'recurringSubscriptionInfo' do
           if reference
-            _, subscription_id, _ = reference.split(";")
+            _, subscription_id = reference.split(";")
```

Comment:
After playing around with it in irb to confirm, I believe this still assigns the second value in the string (ReferenceID). You'll probably need to specify the number of underscores matching the number of values that precede the SubscriptionID on the end.

Reply:
Or just do `subscription_id = reference.split(';')[6]` (IIRC), which I think reads a bit more clearly anyway. (Although TIL Ruby can do destructuring binds, which I hadn't known previously.)

Reply:
👍